Packs/NetQuestOMX/Integrations/NetQuestOMX/NetQuestOMX_test.py

```diff
@@ -0,0 +1,237 @@
+
+    requests_mock.get(f'{BASE_URL}Systems/Slot/{slot_number}/Ipfix/Status/ExportHwm',
+                      json=util_load_json('test_data/ExportPeakFPS.json'))
+
+    requests_mock.get(f'{BASE_URL}Systems/Slot/{slot_number}/Port/{port_number}/'
+                      f'EthernetInterfaces/Status/EthRxTx',
+                      json=util_load_json('test_data/OptimizationStats.json'))
+
+    statistic_types_to_fetch = ["Metering Stats", "Export Stats", "Export Peaks FPS", "Optimization Stats"]
+
+    events = fetch_events(
+        client=net_quest_omx_client,
+        slot_number=slot_number,
+        port_number=port_number,
+        statistic_types_to_fetch=statistic_types_to_fetch
+    )
+
+    assert len(events) == len(statistic_types_to_fetch)
+
+    for event in events:
+        assert event['STAT_TYPE'] in [
+            statistic_type.replace(" ", "")
+            for statistic_type in statistic_types_to_fetch
+        ]
+
+
+def test_get_events(requests_mock, net_quest_omx_client):
+    """
+    Given:
+        - 2 statistic_types_to_fetch
+    When:
+        - Executing get_events function
+    Then:
+        - Ensure number of events as number of statistic_types_to_fetch (event for each type)
+    """
+
+    params = {"slot": "1", "port": "1"}
+    args = {"statistic_types_to_fetch": "Metering Stats,Export Stats"}
+
+    requests_mock.get(f'{BASE_URL}Systems/Slot/{params["slot"]}/Ipfix/Status/Metering',
+                      json=util_load_json('test_data/MeteringStas.json'))
+
+    requests_mock.get(f'{BASE_URL}Systems/Slot/{params["slot"]}/Ipfix/Status/Export',
+                      json=util_load_json('test_data/ExportStats.json'))
+
+    events = get_events(net_quest_omx_client, params, args)
+
+    assert len(events) == 2
+
+
+def test_get_events_invalid_input(net_quest_omx_client):
+    """
+    Given:
+        - invalid inputs -  statistic_types_to_fetch
+    When:
+        - Executing get_events function
+    Then:
+        - Ensure an exception is thrown
+    """
+
+    params = {"slot": "1", "port": "1"}
+    args = {"statistic_types_to_fetch": "Metering ,Export"}
+    with pytest.raises(CommonServerPython.DemistoException):
+        get_events(net_quest_omx_client, params, args)
+
+
+def test_address_list_upload_command(mocker, requests_mock, net_quest_omx_client):
+    """
+    Given:
+        - An entry_id (for a file to upload)
+        - A mocked client
+    When:
+        - Executing netquest-address-list-upload command
+    Then:
+        - Ensure command is not failed and the readable_output as expected
+    """
+    mocker.patch.object(demisto, 'getFilePath', return_value={'path': 'test_data/test_file.txt', 'name': 'test_file'})
+    requests_mock.post(f'{BASE_URL}v1/UpdateService/ImportList/Config', json={})
+    result = address_list_upload_command(client=net_quest_omx_client, args={"entry_id": "AAAAAAaaaa"})
+    assert result.readable_output == "Address list was successfully uploaded"
```

Comment:
add an assertion that the mock was called - here and in other every other test that doesnt rely on the response content